### PR TITLE
Remove Canary install files from the packaging folder

### PR DESCRIPTION
**Pull request by im-konge**
### Type of change

- Removal

### Description

This PR removes the Canary installation files from the `packaging` folder, as agreed in proposal - [86 - Archive Canary project](https://github.com/strimzi/proposals/blob/main/086-archive-canary.md).
The installation files from the `install` folder will be removed as part of the next Strimzi release.
This also removes the step from the Makefile to copy the Canary install files to the release folder.

### Checklist

- [ ] Update install files

### CODE REVIEW

**Comment by im-konge** [comment]
@PaulRMellor should I do something to remove it from the docs? Or will that be removed as part of the next Strimzi release? I saw some mentions in the `deploying` and `overview` folders of the `documentation` folder

**Comment by scholzj** [comment]
> @PaulRMellor should I do something to remove it from the docs? Or will that be removed as part of the next Strimzi release? I saw some mentions in the `deploying` and `overview` folders of the `documentation` folder

There is no such update to the docs as part of the release. So you need to do it here.

**Comment by im-konge** [comment]
> > @PaulRMellor should I do something to remove it from the docs? Or will that be removed as part of the next Strimzi release? I saw some mentions in the `deploying` and `overview` folders of the `documentation` folder
> 
> There is no such update to the docs as part of the release. So you need to do it here.

Cool, thanks for info :) will do it here.

**Review by scholzj — APPROVED** [review]

**Comment by im-konge** [comment]
Good point -> I added a line to the `Changes, deprecations and removals` section, mentioning the proposal and that the project was archived. Thanks @scholzj

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,6 +23,8 @@
   This change matches user expectations.
 * The External Configuration (`.spec.externalConfiguration`) in `KafkaConnect` and `KafkaMirrorMaker2` resources is deprecated and will be removed in the future.
   Please use the environment variables, additional volumes and volume mounts in Pod and container templates instead.
+* Based on the proposal - [86 - Archive Canary project](https://github.com/strimzi/proposals/blob/main/086-archive-canary.md) - the installation files of the Canary project
+  were removed from the repository - as the project was archived.
```
Comment (scholzj) [inline CHANGELOG.md:27]: Maybe we can use this?

```suggestion
* The Strimzi Canary installation files were removed based on the [_Strimzi proposal 086_](https://github.com/strimzi/proposals/blob/main/086-archive-canary.md) as the project was discontinued and archived.
```

Comment (im-konge) [reply CHANGELOG.md]: That's better, thanks

**Pull request merged**